episodes/enabling-technologies-standards.Rmd

```diff
@@ -0,0 +1,699 @@
+
+::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::: instructor
+## Challenge: JSON schema / Form input & validation with JSON schema
+
+The following challenges 4 & 5 will be processed **consecutively in groups of 5 - 6 learners**.
+We made the experience, that changing the group composition for these tasks benefits the overall collaborative atmosphere. 
+
+**Total time:** 30 min
+
+**Implementation:** <br>
+In accordance with challenges 2 & 3, we recommend using a pre-structured shared notes document with the gorups.
+
+**Shared notes:** <br>
+You can find an example Markdown file for the group handouts <a href="../files/exampleGroupHandout.txt" target="_blank">**HERE**</a>. This document is optimized for use in a Hedgedoc document. 
+
+##### <u>Challenge 4: JSON Schema</u>
+
+**Time:** 20 min
+
+**Challenge type:** group activity, production
+
+**Objective:** <br>
+By writing a short excerpt of a JSON schema, the learner get familiar with the schema syntax, gain the ability to read and understand a schema, and get to know some important JSON schema keywords. By writing the schema in a group set-up, the learner experience the discussion process as a crucial part of schema development.
+
+##### <u>Challenge 5: Form input & validation with JSON schema</u>
+
+**Time:** 10 min
+
+**Challenge type:** group activity **OR** individual exploration
+
+**Objective:** <br>
+By downloading and inspecting the final JSON schema, the learners comprehend the complexity a metadata schema can acquire. With the implementation of the schema in the UI of the react-jsonschema-form playground, the learners bring the developed schema into use, get to know a software tool, they can benefit from, and experience relieve of the frustration after Challenge 3.
+
+::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
+
+:::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::: challenge
+## Challenge 4: JSON Schema
+    
+After a couple of researchers upload their JSON metadata records to the project repository it becomes obvious that well-formed JSON metadata describing similar experiments can still be expressed in myriad ways.
+
+Your collaboration decides to develop a metadata schema to standardize metadata records across the project. Consensus is encoded in a JSON Schema.
+
+**Now its your task to help with the subschema for experimental conditions!**
+
+In the following **code block** you see valid JSON metadata that describes experimental conditions as agreed on in the project.
+<br>
+
+```json
+{
+  "experimentalConditions": {
+    "ride": {
+      "rideType": "roller coaster",
+      "rideName": "Flight of the Bat",
+      "location": "Gotham City, New Jersey"
+    },
+    "testPerson": {
+      "sex": "male",
+      "height": 180
+    },
```

Comment:
If the example of the previous episode is updated, this one should probably be also updated.